applications/SwimmingDEMApplication/tests/candelier_tests/ProjectParametersNoHistory.json

```diff
@@ -125,6 +124,33 @@
 "print_VELOCITY_LAPLACIAN_option"            : false,
 "print_VELOCITY_LAPLACIAN_RATE_option"       : false,
 
+"properties": [{
```

Comment:
here "properties" is at the root level of the DEM json. Shouldn't it be "swimming_dem_properties"?

Reply:
Yes, but this will be done in the next PR. There are no swimming_dem_properties (general properties) yet.